apis/s3/src/main/java/org/jclouds/s3/domain/CannedAccessPolicy.java

```diff
@@ -36,29 +38,50 @@
     /**
      * Owner gets FULL_CONTROL. No one else has access rights (default).
      */
-    PRIVATE("private"),
+    PRIVATE,
     /**
      * Owner gets FULL_CONTROL and the anonymous identity is granted READ
      * access. If this policy is used on an object, it can be read from a
      * browser with no authentication.
      */
-    PUBLIC_READ("public-read"),
+    PUBLIC_READ,
     /**
      * Owner gets FULL_CONTROL, the anonymous identity is granted READ and
      * WRITE access. This can be a useful policy to apply to a bucket, but is
      * generally not recommended.
      */
```

Comment:
PUBLIC_READ

Reply:
I don't understand any of your comments.  Can you use complete sentences?